charts/spinnaker/templates/_helpers.tpl

```diff
@@ -81,3 +81,11 @@ Create name of kubeconfig file to use when setting up kubernetes provider
 {{- printf "/opt/kube/%s" .Values.kubeConfig.secretKey  | toString -}}
 {{- end }}
 {{- end }}
+
+{{/*
+Create global template to skip Docker registry enabling validation iff there
```

Comment:
Slightly reworded to:

```
Create a template to skip global Docker registry enabling validation iff there
```

Had a bit more trouble with singular -> plural change. In this case it runs on the `hal config provider docker-registry enable` command which is more like global configuration to enables the usage of one or more docker registries in general. Didn't want to confuse the reader that this would disable validation on all registries, because that it doesn't do. LMKWYT.